static-site/src/util/blogPosts.js

```diff
@@ -36,6 +42,76 @@ export function getBlogPosts() {
   return blogPosts;
 }
 
+/**
+ * Writes RSS2, Atom, and JSON feeds into the `public/blog` directory.
+ */
+export async function generateBlogFeeds() {
+  const feedDir = `${__dirname}/../../public/blog`;
+  const siteURL = "https://nextstrain.org/blog";
+
+  const feed = new Feed({
+    title: "Nextstrain.org Blog",
```

Comment:
_non-blocking_

```suggestion
    title: "Nextstrain Blog",
```

This is the blog of Nextstrain that happens to be on nextstrain.org, not a blog specifically about nextstrain.org